app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/Thumbnail.kt

```diff
@@ -439,6 +444,26 @@ enum class ThumbnailRequestSize(val size: Int) {
     }
 }
 
+@Suppress("MagicNumber")
+private fun ImageRequest.Builder.applyAllSupportedImageDecoders() = apply {
```

Comment:
this extension does not apply all decoders.
But it applies the appropriate decoder based on mimetype. Does that mean another name then? 😄

Reply:
Actually yes will rename it 👍